Makefile

```diff
@@ -11,14 +13,15 @@ run:
 	docker-compose -f docker-compose.test.yml run test
 
 base_spec = spec/base/**/*_spec.rb,spec/addons/docker/**/*_spec.rb
+bootstrap_spec = spec/addons/bootstrap/**/*_spec.rb
```

Comment:
How about we refactor here a bit @rafayet-monon 

```
base_addon_spec = spec/addons/**/*_spec.rb
web_addon_spec = spec/addons/variants/web/**/*_spec.rb
api_addon_spec = spec/addons/variants/api/**/*_spec.rb
base_spec = spec/base/**/*_spec.rb
web_spec = spec/variants/web/**/*_spec.rb
api_spec = spec/variants/api/**/*_spec.rb
...
	if [ $(VARIANT) = web ]; then \
	  bundle exec rspec --pattern="$(base_spec),$(web_spec)$(base_addon_spec),$(web_addon_spec),"; \
	elif [ $(VARIANT) = api ]; then \
	  bundle exec rspec --pattern="$(base_spec),$(api_spec)$(base_addon_spec),$(api_addon_spec),"; \
	fi;
```

Reply:
so it required you move the `boostrap addon` from `.template/addons/boostrap` to `.template/addons/variants/web/boostrap`

Reply:
Or I can do the refactor on another PR, so you just need to  change the path to `.template/addons/variants/web/boostrap` @rafayet-monon

Reply:
The Makefile was refactored in https://github.com/nimblehq/rails-templates/pull/248, so please rebase this PR @rafayet-monon 🙏

Reply:
Added in [d534b0d](https://github.com/nimblehq/rails-templates/pull/239/commits/d534b0d6ffef858976ae1c54b9db07701fbaca71)